lib/core/config/network_config.dart

```diff
@@ -0,0 +1,32 @@
+part of '../core.dart';
+
+/// Manage all url creation and building
+// ignore_for_file: avoid_classes_with_only_static_members
+class NetworkConfig {
```

Comment:
We could have it outside a class just as functions instead of methods

Reply:
Well, it's just a function, just like any other utility functions.

You can add the SettingsRepository and get the `baseApiEndpointUrl` from it.